### ISSUE TITLE
A degradation between 9.1.3 and 9.1.4 versions (with one-of)

### ISSUE THREAD

**Issue by otlg** [rapi-doc/RapiDoc#753]
Hi,

We tried to update RapiDoc to the latest version (9.2.0) mainly because of vulnerability issues in its dependencies. 
However, we are faced with the issue where examples (both request body and response) are not presented correctly. 

We investigated it a little and found that issue was introduced in the 9.1.4 version, you can also check a simple example at https://codesandbox.io/s/rapidoc-9-1-4-issue-z4xhjs?file=/src/App.js

In the **9.1.4** version, the YAML is not presented properly and UI does not present examples based on one-of. We get an empty dropdown list.
![image](https://user-images.githubusercontent.com/43635343/169994985-c54c5f65-89b6-4383-86e0-38c50fa4b111.png)

The 9.1.3 presents it correctly:
![image](https://user-images.githubusercontent.com/43635343/169995193-93823516-8207-4235-9974-6d3adcf6e896.png)

A simple example of a response (but same happens for request body as well)

```yaml
openapi: 3.0.1
info:
  title: Test unorder schemas
  description: |-
    Should not report {recursive}
  version: "1.0"
paths:
  /api/test:
    get:
      responses:
        "200":
          description: Status OK
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Parent"
components:
  schemas:
    Child1:
      type: object
      allOf:
        - $ref: "#/components/schemas/Parent"
    Child2:
      type: object
      allOf:
        - $ref: "#/components/schemas/Parent"
        - type: object
          properties:
            field3:
              type: string
    Parent:
      required:
        - type
      type: object
      properties:
        name:
          type: string
        type:
          type: string
      discriminator:
        propertyName: type
        mapping:
          one: "#/components/schemas/Child1"
          tow: "#/components/schemas/Child2"
          three: "#/components/schemas/Child3"
          four: "#/components/schemas/Child4"
      oneOf:
        - $ref: "#/components/schemas/Child1"
        - $ref: "#/components/schemas/Child2"
        - $ref: "#/components/schemas/Child3"
        - $ref: "#/components/schemas/Child4"
    Child3:
      type: object
      allOf:
        - $ref: "#/components/schemas/Parent"
        - type: object
          properties:
            field1:
              type: string
              enum:
                - A1
                - A2
    Child4:
      type: object
      allOf:
        - $ref: "#/components/schemas/Parent"
        - type: object
          properties:
            field2:
              type: string
              enum:
                - B1
                - B2
```

Thanks,
Oleg.

**Comment by otlg** [comment]
In the provided example, the link to the file is broken, but its content is the same I added to the issue.

**Comment by mrin9** [comment]
this should be fixed in `9.3.2` release. Closing this, if you still find its broken, please create a new issue and link it to this one

**Comment by otlg** [comment]
Thanks, looks good.
